Assets/MixedRealityToolkit.SDK/Features/Utilities/StabilizationPlaneModifier.cs

```diff
@@ -0,0 +1,390 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [Serializable]
+    public struct StabilizationPlaneOverride
+    {
+        public Vector3 Center;
+        public Vector3 Normal;
+    }
+
+    /// <summary>
+    /// StabilizationPlaneModifier handles the setting of the stabilization plane in several ways.
+    /// Using StabilizationPlaneModifier with override DepthLSR which is automatically enabled via depth buffer sharing
```

Comment:
Maybe call out SetFocusPointForFrame() api since that is heart of reason for this component. Also re-write this line to Stabilization PlaneModifer can be used in conjunction with DepthLSR which is enabled by Depth Buffer Sharing etc 

 Although on HL1 I think the stabilization plane can be set in RS5 via the depth buffer sharing and I'm not sure which one takes precedence...

Reply:
SetFocusPoint takes precedence over depth buffer sharing.